scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -449,6 +455,14 @@ class BigQueryClient private (private val projectId: String,
     }
   }
 
+  private def getUdfResource(): Option[java.util.List[UserDefinedFunctionResource]] = {
+    BigQueryClient.udfResourceUri.map { uri =>
```

Comment:
So for standard SQL it'll work out of the box without any Scio modification right? In that case I'd prefer not adding anything awkward to support legacy SQL then.